### PR TITLE
add bootcamp to projects list

### PR DESCRIPTION
**Pull request by SriNandan33**
I have added my flask application to the projects list. Please review it and let me know if I need to make any changes. Thanks you so much for creating such a platform that makes opesource contributions easy for beginners like me.

### CODE REVIEW

File: _data/projects/bootcamp.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+desc: A mini social networking application using Flask
+site: https://github.com/SriNandan33/bootcamp
+tags:
+- oss
+- web
+- flask
+- python
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/up-for-grabs/up-for-grabs.net/labels/up-for-grabs
```
Comment (shiftkey) [inline _data/projects/bootcamp.yml]: This link should point to a label in https://github.com/SriNandan33/bootcamp that's associated with issues which are ideal for new contributions. I see you have `Hacktoberfest` and `good first issue` in use - did you have a preference?

Comment (SriNandan33) [reply _data/projects/bootcamp.yml]: Thanks you so much @shiftkey , I have updated changed the lable to Hacktoberfest and updated the link as well .

I have a question, should I create a new PR everytime I open an issue on my public repo?

Comment (shiftkey) [reply _data/projects/bootcamp.yml]: > I have a question, should I create a new PR everytime I open an issue on my public repo?

@SriNandan33 no need to do this - we have a GitHub Action that regenerates the stats for all projects daily. You can see an example of that in action here: #1385

**Review by shiftkey — APPROVED** [review]
Thanks @SriNandan33!

**Pull request merged**